ui/src/components/LinkProfilePopup.vue

```diff
@@ -35,27 +37,20 @@ defineProps<{
   agentPubKey: Uint8Array;
 }>();
 
-const PROFILE_SHOW_HIDE_DELAY = 400; // in ms
```

Comment:
Nice that the popup doesn't jump around anymore. I had opened an issue for that a long time ago https://github.com/quasarframework/quasar/issues/13517
It's never been looked at =(

Could be that we want a delay on hover, but I'd say for the moment let's try without it.